app/views/registrations/new.html.erb

```diff
@@ -0,0 +1,118 @@
+        </ul>
+      </p>
+      <p>4. <strong>Privacy Policy</strong><br>
+        Your use of the Service is also governed by our Privacy Policy, which can be found at [link to Privacy Policy]. By using the Service, you consent to the collection and use of your personal data in accordance with our Privacy Policy.
+      </p>
+      <p>5. <strong>Account Registration</strong><br>
+        To use certain features of the Service, you may need to create an account. You agree to provide accurate, current, and complete information during the registration process and to update your information as necessary. You are responsible for maintaining the confidentiality of your account credentials and for all activities under your account.
+      </p>
+      <p>6. <strong>Intellectual Property</strong><br>
+        The Service and all its content, features, and functionality (including but not limited to text, graphics, logos, images, and software) are owned by or licensed to [Your Company Name] and are protected by intellectual property laws. You may not use, reproduce, or distribute any content from the Service without our prior written consent.
+      </p>
+      <p>7. <strong>Prohibited Activities</strong><br>
+        You agree not to engage in any of the following prohibited activities:
+        <ul class="list-disc pl-5">
+          <li>Unauthorized access to or use of the Service.</li>
+          <li>Uploading or distributing malware, viruses, or any other harmful code.</li>
+          <li>Engaging in fraudulent, deceptive, or abusive behavior.</li>
+        </ul>
+      </p>
+      <p>8. <strong>Termination</strong><br>
+        We may suspend or terminate your access to the Service at any time, without notice, for any reason, including if we believe you have violated these Terms.
+      </p>
+      <p>9. <strong>Limitation of Liability</strong><br>
+        To the fullest extent permitted by law, [Your Company Name] will not be liable for any indirect, incidental, special, consequential, or punitive damages arising from your use of the Service, even if we have been advised of the possibility of such damages.
+      </p>
+      <p>10. <strong>Governing Law</strong><br>
+        These Terms are governed by and construed in accordance with the laws of [Insert Jurisdiction], without regard to its conflict of law principles.
+      </p>
+      <p>11. <strong>Dispute Resolution</strong><br>
+        Any disputes arising out of or relating to these Terms or the use of the Service will be resolved through binding arbitration in [Insert Jurisdiction], and you consent to the exclusive jurisdiction and venue of the courts in that location.
+      </p>
+    </div>
+
+    <div class="flex justify-center mt-4">
+      <button id="close-modal" class="px-6 py-2 bg-teal-600 text-white font-semibold rounded-lg hover:bg-teal-700 transition">
+        I Understand
+      </button>
+    </div>
+  </div>
+</div>
```

Comment:
Bu modal kodu burada olmamalı. Bunu partial view ile dahil edelim. Ayrıca metinleri html içine yazmıyoruz bunlar için bir translation file oluşturup oraya alalım.